crates/store/src/db/migrations/001-init.sql

```diff
@@ -0,0 +1,67 @@
+CREATE TABLE
+    settings
+(
+    name  TEXT NOT NULL,
+    value BLOB,
+
+    PRIMARY KEY (name),
+    CONSTRAINT settings_name_is_not_empty CHECK (length(name) > 0)
+) WITHOUT ROWID;
```

Comment:
@hackaugusto with `STRICT` flag it won't work as I expected. In order to make data human-readable and simplify the code, I use native SQLite feature for writing different types of values to the same field. I prefer strong typing, but for this case it's very convenient.

Reply:
I don't follow the rationale, can you give an example? (also, not that big of a deal, if you say it is better like this, fine with me)

Reply:
I use settings for storing schema version (integer) and migration hash (string). I had several options for writing this into a table:
1. Write values as is (don't use `STRICT` flag). This is current approach.
2. Store values as binary encoded BLOB (in this case we would need to deserialize values from binary in order to examine them).
3. Store values as STRING (this is also human-readable, but we would need to require implementation of `FromStr`/`ToString` traits from data, serialize/parse and map errors).

According to the current usage, I think, the 3rd solution is also good, so I can rewrite `Settings` to 3rd approach.

Reply:
I tried 3rd approach and it's not so good, as I expected due to `FromStr` error type, which is associated type (but there is also error from rusqlite, which is also should be returned). Thus, it would be easier to parse string outside of settings. Now I think, the 3rd approach is worse, than the 1st.

Reply:
Ah, I see, I was under the impression that converting to from/bytes would be easy to implement.

If that is the case, then maybe lets follow the approach [documented here](https://www.sqlite.org/flextypegood.html#attribute_tables), and instead of calling the type `BLOB` lets call `ANY` and add a comment explaining it?

Reply:
@hackaugusto I didn't know about `ANY` datatype, thank you! Fixed it!